src/main/java/ua/bank/bankservice/controllers/DataInject.java

```diff
@@ -0,0 +1,35 @@
+package ua.bank.bankservice.controllers;
+
+import java.time.LocalDate;
+import java.util.Set;
+import javax.annotation.PostConstruct;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Component;
+import ua.bank.bankservice.model.Role;
+import ua.bank.bankservice.model.User;
+import ua.bank.bankservice.service.RoleService;
+import ua.bank.bankservice.service.UserService;
+
+@Component
+@AllArgsConstructor
+public class DataInject {
+    private final RoleService roleService;
+    private final UserService userService;
+
+    @PostConstruct
+    private void init() {
+        Role user = new Role();
+        user.setRoleName(Role.Roles.USER);
+        roleService.create(user);
+        Role admin = new Role();
+        admin.setRoleName(Role.Roles.ADMIN);
```

Comment:
Such enum naming leads to confusion, consider refactoring to RoleType or smth else